### ISSUE TITLE
[security] Whats the point of Screen authentication

### ISSUE THREAD

**Issue by fjaeckel** [concerto/concerto#1369]
When receiving a list of all active authentication tokens through an **unauthenticated** request to the `/screens.{xml,json}`

Reproduce: `curl http://concerto/screens.json`

Vector: Taking over screens, the player of a screen, leaking all information to unauthenticated attackers.

The fix must include either removing `.{xml,json}` from the scope of this or adding a `auth!`:
https://github.com/concerto/concerto/blob/master/app/controllers/screens_controller.rb#L11

**Comment by augustf** [comment]
@mikldt Does this seem ok to you? I'm unsure if we're depending on this screen list in bandshell or on the player side generally...

**Comment by rom650** [comment]
Any news ?

**Comment by sundhaug92** [comment]
Yeah, this is kinda bad. One alternative would be to have the authentication-tokens not be visible for users that aren't logged in, but I think requiring auth is the best alternative. @mfrederickson?

**Comment by mfrederickson** [comment]
I'm not well versed in the security side of the screens, but is this still an issue-- because when I have "Public Concerto Access" unchecked, the curl response is empty.

**Comment by sundhaug92** [comment]
@mfrederickson What's the point of authentification tokens that can be gained without any authentification?

**Comment by mfrederickson** [comment]
@sundhaug92 oh, I agree with you. With that checkbox unchecked, and not logged in, you cannot see them.
